cmd/introspect/main.go

```
@@ -1252,7 +1254,15 @@ func parseBamlSourceDir(dir string) *bamlConfig {
 		if readErr != nil {
 			return nil // skip unreadable files
 		}
-		parseBamlFile(cfg, string(data))
+		file, parseErr := bamlparser.ParseBytes(path, data)
+		if parseErr != nil {
+			// The new parser is intentionally permissive (Codex scoping Q2/Q6);
+			// any real-world parse error is a parser bug to be fixed by adding
+			// a fixture rather than swallowing here. Match the old line walker's
+			// silent-on-unreadable posture so generation stays best-effort.
+			return nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 1257) but it returns nil

(nilerr)